File: 백준/Bronze/4344. 평균은 넘겠지/평균은 넘겠지.py
C = int(input())

for i in range(C):
    num = list(map(int, input().split()))
    numK = num.pop(0)
    number = 0
    aver_num = 0
    for j in num:
        number += j
    sum_number = number / numK
    for k in num:
        if k > sum_number:
            aver_num += 1
    
    result = aver_num / numK * 100
    print('{0:.3f}%'.format(result))

File: 백준/Bronze/4344. 평균은 넘겠지/평균은 넘겠지.js
const fs = require("fs");
const input = fs.readFileSync("/dev/stdin").toString().split("\n");

const C = Number(input[0]);
for (let i = 1; i <= C; i++) {
  // 배열
  const array = input[i].split(" ").map(Number);

  // 배열[i]의 평균
  const average = array.slice(1).reduce((acc, cur) => acc + cur, 0) / array[0];

  let count = 0;
  for (let j = 1; j <= array[0]; j++) {
    if (average < array[j]) {
      count += 1;
    }
  }

  let result = (count / array[0]) * 100;
  console.log(`${result.toFixed(3)}%`);
}
